lib/app/person.ex

```diff
@@ -0,0 +1,39 @@
+defmodule App.Person do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias App.{Item, Repo, Tag}
+  alias __MODULE__
+  # https://hexdocs.pm/phoenix/Phoenix.Param.html
```

Comment:
I find the docs for `Param` pretty useless. Wish there was a standalone example. 💭